<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>   MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateProfileBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Update Profile </string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/oneuser.png</normaloff>../IMAGES/oneuser.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="allReservationsBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>All Reservations</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/Reservations.jpeg</normaloff>../IMAGES/Reservations.jpeg</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="showAllUsersBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show Users</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/users1.png</normaloff>../IMAGES/users1.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="orderHistoryBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Order History</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/order history.png</normaloff>../IMAGES/order history.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="reservationBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reservation</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/reservationsadd.png</normaloff>../IMAGES/reservationsadd.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="allOrdersBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>All Orders </string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/ALLORDERS.jpg</normaloff>../IMAGES/ALLORDERS.jpg</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/foodmenupic.png</normaloff>../IMAGES/foodmenupic.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="dashboardBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/dashboard.png</normaloff>../IMAGES/dashboard.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../IMAGES/EXITbutton.png</normaloff>../IMAGES/EXITbutton.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="display_graph_widget" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>841</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome Name of User</string>
   </property>
  </widget>
  <widget class="QComboBox" name="menuCategoryFilter">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
